SonarQube.Bootstrapper/Program.cs

```diff
@@ -11,27 +11,34 @@
 
 namespace SonarQube.Bootstrapper
 {
-    internal static class Program
+    public static class Program
     {
-        private const int ErrorCode = 1;
+        public const int ErrorCode = 1;
 
         private static int Main(string[] args)
         {
             var logger = new ConsoleLogger();
+            BuildAgentUpdater updater = new BuildAgentUpdater();
 
+            return Execute(args, updater, logger);
+        }
+
+        public static int Execute(string[] args, IBuildAgentUpdater updater, ILogger logger)
```

Comment:
Moved the operative code to a method that tests can call directly, supplying appropriate test/mock instances as arguments.